rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -454,30 +442,30 @@ extern "C" RocalTensor ROCAL_API_CALL rocalFishEye(RocalContext context, RocalTe
                                                    RocalTensorLayout output_layout = ROCAL_NONE,
                                                    RocalTensorOutputType output_datatype = ROCAL_UINT8);
 
-/*! \brief Applies vignette effect on images.
+/*! \brief Accepts U8 and RGB24 input.
```

Comment:
I think this briefing should be retained and 
Accepts U8 and RGB24 input should be added as part of the /note